config/locales/en.yml

```diff
@@ -63,6 +63,7 @@ en:
             multiple_channel_integrations: Specifying multiple channel integrations
               in requirements.json is not supported.
             too_many_oauth_parameters: "Too many parameters with type 'oauth': one permitted"
+            oauth_parameter_required: "At least one parameter with the type 'oauth' is required"
```

Comment:
Only one `oauth` parameter is currently allowed. "At least one parameter" might imply that it is possible to have more than one perhaps?